avro-serializer/src/main/java/io/confluent/kafka/serializers/KafkaAvroDecoder.java

```diff
@@ -28,12 +28,17 @@ public KafkaAvroDecoder(SchemaRegistryClient schemaRegistry) {
     this.schemaRegistry = schemaRegistry;
   }
 
+    public KafkaAvroDecoder(SchemaRegistryClient schemaRegistry, boolean useSpecificDecoder) {
```

Comment:
I think this is useful for the tests actually. Otherwise we can't pass in a local (or mock) schemeRegistry in for the test. The alternative is we could modify the properties so it can take a schemaRegistryFactory or similar?